tools/openms/.shed.yml

```diff
@@ -0,0 +1,12 @@
+name: openms
+owner: galaxyp

```

Comment:
We want to give people the opportunity to just pick a few tools put of 120. We still get one meta-package that will contains a link to all others if you want to get all.